input/pagecontent/changes.md

```diff
@@ -158,7 +158,8 @@ This change log documents the significant updates and resolutions implemented fr
   - removed the required binding to Metric Body Length Units value set from Observation.valueQuantity.code to allow the FHIR standard profile binding, and added the Metric Body Length Units value set as a candidate additional binding [FHIR-44787](https://jira.hl7.org/browse/FHIR-44787)
   - removed invariant au-core-obs-01 (duplicates invariant vs-1) [FHIR-46486](https://jira.hl7.org/browse/FHIR-46486)
 - Corrected the search parameter for gender identity to be `gender-identity` not `patient-gender-identity` [FHIR-45057](https://jira.hl7.org/browse/FHIR-45057).
-- Changed the following Patient search parameters: 
+- Clarified the use of multipleAnd and comparators on AU Core date search parameters by appending the following to multipleAnd constraints: " and if multipleAnd is supported, SHALL support the search comparators 'gt', 'lt', 'ge', 'le'." [FHIR46309](https://jira.hl7.org/browse/FHIR-46309)
```

Comment:
Suggest making this a bit simpler, something like
- Updated AU Core date search parameters to clarify that if multipleAnd is supported, the search comparators 'gt', 'lt', 'ge', 'le' SHALL also be supported. [FHIR46309](https://jira.hl7.org/browse/FHIR-46309)

Reply:
shortened the wording